dockerfiles/bloodhound.Dockerfile

```diff
@@ -26,48 +26,27 @@ ARG AZUREHOUND_VERSION=v2.1.9
 FROM --platform=$BUILDPLATFORM docker.io/library/node:20-alpine AS deps
 ARG version=v999.999.999
 ARG checkout_hash=""
-ENV PYTHONUNBUFFERED=1
+ENV SB_LOG_LEVEL=debug
 ENV VERSION=${version}
 ENV CHECKOUT_HASH=${checkout_hash}
 WORKDIR /bloodhound
 
-RUN apk add --update --no-cache python3 git go
-RUN find /usr/lib -name EXTERNALLY-MANAGED -delete
-RUN python3 -m ensurepip
-RUN pip3 install --no-cache --upgrade pip setuptools
+RUN apk add --update --no-cache git go
 
 COPY . /bloodhound
+RUN go run github.com/specterops/bloodhound/packages/go/stbernard deps
 
 ########
-# Build UI
+# Build
 ################
-FROM deps AS ui-builder
-
-WORKDIR /bloodhound/packages/javascript/bh-shared-ui
-RUN yarn install
-RUN yarn build
-
-WORKDIR /bloodhound/packages/javascript/js-client-library
-RUN yarn install
-RUN yarn build
-
-WORKDIR /bloodhound
-RUN python3 packages/python/beagle/main.py build bh-ui -v
-
-########
-# Build API
-################
-FROM deps AS api-builder
+FROM deps AS builder
 ARG TARGETOS
 ARG TARGETARCH
-ENV GOOS=${TARGETOS}
-ENV GOARCH=${TARGETARCH}
 ENV CGO_ENABLED=0
+ENV VERSION=${version}
```

Comment:
We may want to use a more targeted ENV var just to prevent possible collisions in the future. This would also mean changing what St Bernard looks for. Perhaps something like SB_VERSION since SB-specific env vars are supposed to be prefixed.